### DISCUSSION TITLE
Highlighting a group of measures of a specific musical line/instrument.

### DISCUSSION THREAD

**Discussion by ammatwain** [opensheetmusicdisplay/opensheetmusicdisplay#1448] [Q&A] [answered]
Hello everyone! For my self-taught purposes (being self-taught is always a paradox, one must be knowledgeable and ignorant at the same time, in my case, the latter often prevails...), I would need a result like this:

![SPARTITO](https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/assets/8464862/a46eb88b-6450-4b64-8a4d-b4d473b29f41)

Namely, highlighting a group of measures of a specific musical line/instrument. I took a look at the color system and I think I understood that it's possible to modify the colors of the notes. For the result I have in mind, I would need something like: "always use gray, but when you're on this line and this measure, then use black."

Is there something in OSMD that guides me toward this result, or do I need to get involved with the graphical representation process?

Best regards!

Amedeo

**Comment by sschmidTU** [comment, marked as answer]
This would be quite difficult, you'll need to get involved, yes.
An alternative would be using multiple cursors with cursor type 3 (whole measure), one for each measure.
But if you want to only highlight one part of an instrument, you'll probably have to modify the cursor code.
<img width="151" alt="image" src="https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/assets/33069673/72e5b13f-bcb5-4f87-975b-a1e02f5632ec">
`osmd.cursor.cursorOptions.type = 3`
(you can also set color and opacity)

(closing for now, feel free to reopen or comment again later)

**Reply by ammatwain** [reply]
Thank you, @sschmidTU , that's an alternative I hadn't considered, but it would still achieve my goal
